bin/firebase-bolt

```diff
@@ -106,10 +107,14 @@ function main() {
       log("Could not read file: " + inFile);
       process.exit(1);
     }
-    writeTranslation(outFile, data);
+    writeTranslationWithImports(outFile, data, inFile);
   });
 }
-
+/**
+* Depricated: Use writeTranslation still needs to be maintained as we have the
+* ability to write the translation from STDIN not just the file system.
+* ??? Global modules should still be supported from std in.
+*/
```

Comment:
I would think that if you read from `STDIN` then you would just resolve relative to the current directory instead of the file location. Thoughts? I'm not sure what the best thing for global imports would be... Maybe they should look in global node packages only?

Reply:
Yeah, I didn't really think of std in as a use case. Not sure if this is required though. See comments below about global modules.

Reply:
At least we should gracefully handle errors if people try and do this, but having these two code paths isn't great either.

Reply:
I think throwing a deprecation warning is appropriate but I wouldn't remove it unless there is a major version bump (backward compatability must be maintained). So I would add this as an open issue to remove old behaviour in the next major version bump. 

So 2 suggestions, I will add the deprecation warning and can add the deprecation issue for the next major milestone. (you might have to create the next major milestone tag).